lib/registrant.rb

```diff
@@ -0,0 +1,28 @@
+class Registrant
+    attr_reader :age,
+                :license_data,
+                :name,
+                :permit
```

Comment:
Careful here, we don't need a method called `.permit` so we don't want to write a method we don't need.